### ISSUE TITLE
Suffle button (or Random sorting)

### ISSUE THREAD

**Issue by cyh76507707** [Steemhunt/web#194]
Suggested by librarian (geekgirl)
> Create a "random" view where posts will appear randomly regardless the rank and time posted and refreshed every 5 or 10 minutes. Perhaps also incentivize curation from "random" view.

**Comment by cyh76507707** [comment]
**[Shuffle Button]**
- Located on top of the today's chart
- When a user click the button, on the left view the user will see the hunt post in a random suggestion
- On mobile view there's a floating fixed button for shuffle
- Users can explore hunting posts by clicking shuffle buttons constantly

**Comment by sydneyitguy** [comment]
After some investigation, it seems we have minimized voting pool's influence successfully by adapting Diversity  Score. However, the quality of the ranking board hasn't been improved at all. Why?

I think it's combination of 2 reasons:

1. We have more than 50 unverified posts everyday rolled over from yesterday's ranking. So some posts already have 10+ votings from the beginning of the ranking.
2. A lot of people just vote on ANY posts, doesn't matter who / what the post is. They just want to get some HUNT rewards for voting contribution. As a result, they just vote from the top to bottom, maybe 5-20 posts every  day, absolutely no care about the content or author.

I had a look for few days since ABV + DS, and top rank never changed until the end of the ranking competition.

In conclusion, we may need a direct solution for this problem, instead of "encouraging people to discover more". Just randomize the ranking board for Today's section.

**Comment by sydneyitguy** [comment]
We will do some experiment on our ranking board (Today's section) with randomized sorting option as a default.

Even though we have successfully minimized the voting pool's influence on the ranking board by adapting Diversity Score, the quality of the ranking board hasn't been improved much. I investigated on the  voting data for a while and concluded that:

A lot of people just vote from the top to bottom, absolutely don't care about the content, because they just want to get HUNT tokens by spending their voting power. As a result, top ranks are pretty much fixed after 1-2 hours past the midnight, and doesn't really represent  the quality  of hunts.

To fix this issue, we have decided to change the default sort options of Today's  ranking board from "Hunt Score" to "Random".
If you want to see the actual  ranking, you can still switch it to "Hunt Score".
